glide.yaml

```diff
@@ -71,13 +71,13 @@ import:
 - package: github.com/kubernetes/repo-infra
   version: 4c4ae2e2e6994e887da16f3c31f495e10062b810
 - package: k8s.io/kubernetes
-  version: ebb8d6e0fadfc95f3d64ccecc36c8ed2ac9224ef # client-go origin version
+  version: 4bc5e7f9a6c25dc4c03d4d656f2cefd21540e28c # kubernetes 1.7.6
 - package: k8s.io/client-go
-  version: e356aa2e77ab4a5914c216c12ba14cce25a25ab0 # in sync with kubernetes 1.7.0
+  version: 031a3824a2d20f74ef80e43724a189799f1a1c42 # in sync with kubernetes 1.7.5-beta.0 // TODO update once 1.7.6 is synced
```

Comment:
see issue https://github.com/kubernetes/test-infra/issues/4080
sync from monorepo to client-go (and other synced repos) seems to be broken

Reply:
sync bot was another concern. everytime I hear about the sync, I hear about how it's broken.